### PR TITLE
[AGENT-5587] Update NeMo Inference Server env 

### PR DESCRIPTION
**Pull request by alexromanyuk**
## Summary
- NeMo changed image location to `ohlfw0olaadg/ea-participants/nim_llm`
- Fixed NeMo functional test's model location (changed from NGC registry to S3 location, which has re-compiled model with A10 support)
- Updated docs for NeMo model example

### CODE REVIEW

File: model_templates/gpu_nemo_tensorrt_llm_textgen/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
+## NeMo Inference Server Text Generation Template
+
+This text generation example uses Llama-2-7b model pre-compiled for A100 GPU.
+
+## Requirements
+- NeMo Inference Models currently only available via Early Access program:
```
Comment (elatt) [inline model_templates/gpu_nemo_tensorrt_llm_textgen/README.md:6]: I don't think this is true.

File: model_templates/gpu_nemo_tensorrt_llm_textgen/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+https://developer.nvidia.com/nemo-llm-service-early-access/join
+- Generate a valid NVIDIA NGC Registry API Key:
+https://docs.nvidia.com/launchpad/ai/base-command-coe/latest/bc-coe-docker-basics-step-02.html#optionally-generate-an-ngc-key
+API Key is used to build a docker image and pull LLM models from NGC Registry.
+- Install NGC CLI tool https://org.ngc.nvidia.com/setup/installers/cli
+- Most of NVIDIA models are pre-compiled for TensorRT-LLM support on A100 / V100 GPUs. See the official documentation:
```
Comment (elatt) [inline model_templates/gpu_nemo_tensorrt_llm_textgen/README.md:12]:
```suggestion
- Most of NVIDIA models are pre-compiled for TensorRT-LLM support on A100 / H100 GPUs. See the official documentation:
```

**Review by elatt — APPROVED** [review]
Fine for now to use the early access stuff but when we get access to GA repo, we need to update this and point users there.

File: model_templates/gpu_nemo_tensorrt_llm_textgen/README.md (hunk 1 above)
Comment (elatt) [reply model_templates/gpu_nemo_tensorrt_llm_textgen/README.md:6]: There are a lot of other changes in this PR so probably good to get it merged mostly as-is and update early-access stuff once we get access ourselves.

**Review by rvorobii — APPROVED** [review]

File: model_templates/gpu_nemo_tensorrt_llm_textgen/README.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+## NeMo Inference Server Text Generation Template
+
+This text generation example uses Llama-2-7b model pre-compiled for A100 GPU.
+
+## Requirements
+- NeMo Inference Models currently only available via Early Access program:
+https://developer.nvidia.com/nemo-llm-service-early-access/join
```
[the remark this answers is not in the document]
Comment (elatt) [reply model_templates/gpu_nemo_tensorrt_llm_textgen/README.md:7]: In a follow on-PR

**Comment by yakov-g** [comment]
jenkins run test_functional_general test_e2e_training_model_templates test_e2e_inference_model_templates test_e2e_drop_in_envs

**Pull request merged**